templates/app-pod-yamls/cirros.yaml

```diff
@@ -0,0 +1,42 @@
+kind: 'DeploymentConfig'
+apiVersion: 'v1'
```

Comment:
Please move apiVersion to be first for consistency

Reply:
ack, will update it.